doc/api/errors.md

```diff
@@ -659,6 +659,13 @@ Used when the native call from `process.cpuUsage` cannot be processed properly.
 Used when an invalid value for the `format` argument has been passed to the
 `crypto.ECDH()` class `getPublicKey()` method.
 
+<a id="ERR_CRYPTO_ECDH_INVALID_PUBLIC_KEY"></a>
+### ERR_CRYPTO_ECDH_INVALID_PUBLIC_KEY
+
+Used when an invalid value for the `key` argument has been passed to the
```

Comment:
Nit: There's a PR in to remove `Used when` from all these messages. I'd prefer this go in as something like this:

> An invalid value for the `key` argument was passed to the...

Reply:
Done :) 
But I want to understand the rationale behind the rule. I will help me a lot.
Thanks in advance.

Reply:
It's to reduce wordiness and make the document more scannable. There were some pretty egregious examples, like:

> The 'ERR_INVALID_CURSOR_POS' is thrown specifically when a cursor on
a given stream is attempted to move to a specified row without a specified
column.

That will be shortened to:

> A cursor on a given stream cannot be moved to a specified row without
a specified column.

PR is https://github.com/nodejs/node/pull/16954

Reply:
xD nice example. 
Understood.